IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -12,6 +14,13 @@
 DEFAULT_OPEN_SEES_PATH = "OpenSees"
 DF_INDEX_NAME = "component"
 
+TIME_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+class time_type(Enum):
+    start_time = 0
+    end_time = 1
```

Comment:
If these aren't being imported then just having them as globals is fine.
Given how you're using them (for the name only) having an entire enum seems like overkill.

Reply:
the enum is used in the verify_adv_im.py in gmsim_workflow.

Reply:
Ok, they are being imported by the other PR. This is fine

Reply:
We're trying to limit importing from IM_calc(and Pre-processing among others) to other repos - for circular dependancies. Perhaps this enum is better off in qcore / slurm workflow? (But haven't read much of the PR yet)